src/NServiceBus.Core/Transports/Msmq/MsmqDequeueStrategy.cs

```diff
@@ -154,11 +156,11 @@ bool QueueIsTransactional()
         {
             try
             {
-                return queue.Transactional;
+                return receiveQueue.Transactional;
             }
             catch (Exception ex)
             {
-                var error = string.Format("There is a problem with the input queue: {0}. See the enclosed exception for details.", queue.Path);
+                var error = $"There is a problem with the input queue: {receiveQueue.Path}. See the enclosed exception for details.";
```

Comment:
Except for those string.Format's... I commented on the wrong lines. String interpolation is fine for those but the diff grows for no real reason.